Documentation/gettingstarted/kubeproxy-free.rst

```diff
@@ -1148,6 +1153,13 @@ a fixed cookie value as a trade-off. This makes all applications on the host to
 select the same service endpoint for a given service with session affinity configured.
 To disable the feature, set ``config.sessionAffinity=false``.
 
+For users who run with kube-proxy (i.e. with Cilium's kube-proxy replacement
+disabled), the ClusterIP service loadbalancing when a request is sent from a pod
+running in a non-host network namespace is still performed by the ``bpf_lxc``
+(until `GH#16197<https://github.com/cilium/cilium/issues/16197>`__ is fixed).
+For this case the session affinity support is disabled by default. To enable the
+feature, set ``config.sessionAffinity=true``.
+
```

Comment:
I think we also need to document the kernel requirement in https://docs.cilium.io/en/latest/operations/system_requirements/.